src/app/modules/convai-checker/convai-checker.component.ts

```diff
@@ -243,7 +243,7 @@ export class ConvaiCheckerComponent implements OnInit, OnChanges {
   // TODO: Consider using a CSS selector for this instead, for better
   // specificity.
   @HostListener('document:input', ['$event'])
-  private _handleInputEvent(event: InputEvent) {
+  handleInputEvent(event: InputEvent) {
```

Comment:
Why no longer private? Is it a newer lint thing?

Reply:
This was causing errors when running ng-packagr (I'm guessing it's from some new lint).